src/redux/userActions.js

```diff
@@ -0,0 +1,9 @@
+export const actions = ["user_action"];
```

Comment:
move types to another file and export a key = string. `export const USER_ACTION = 'USER_ACTION';`